### PR TITLE
Clarify blocking of extensions

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR fixes https://github.com/MicrosoftDocs/edge-developer/issues/2436 (Interference of BlockExternalExtensions is unclear)

Rendered article section for review:
* https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options?branch=pr-en-us-2564#blocking-of-extensions [[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/blocking-extensions/microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md#blocking-of-extensions)]
If a link in that section is 404, in Address bar, double-click "review" in https://review.learn.microsoft.com, press Delete, press Enter.

AB#44404774

### CODE REVIEW

File: microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md [hunk 1]
```diff
@@ -151,6 +151,19 @@ See also:
 * [Manifest file format for extensions](../getting-started/manifest-format.md)
 
 
+<!-- ====================================================================== -->
+## Blocking of extensions
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md:155]: C.D. -- is this section heading good?  Should this section be added to this article (which user filed Issue on)?

issue #2436 (Interference of BlockExternalExtensions is unclear)

File: microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md [hunk 2]
```diff
@@ -151,6 +151,19 @@ See also:
...
+
+Question:
+It is not clear if `BlockExternalExtensions` has to be set to `disabled` or `unconfigured` as a prerequisite.  Or if extensions and their origin configured in `ExtensionInstallForcelist`, `ExtensionInstallSources`, or `ExtensionSettings` take precedence over `BlockExternalExtensions=Disabled`.
+
+Answer: 
+Setting the `BlockExternalExtensions` policy to `Not Configured` or `Disabled` won't have any effect.  If the `BlockExternalExtensions` policy is set to `enabled`, external extension installation (by third party software) won't be allowed. The policies for force install list, block list, and extension settings are used when enterprises want to install or block specific extensions on the client machines.
+
+See [BlockExternalExtensions](/deployedge/microsoft-edge-policies#blockexternalextensions) in _Microsoft Edge - Policies_.
+<!-- todo: check spelling of option names etc. -->
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md:164]: C.D. -- I'm looking for the strings to check spelling & capitalization.  What's the exact URL(s) (maybe ending with # anchor) that confirms the spelling of the following strings/values?  

I'm trying to find those values/strings near:
https://learn.microsoft.com/deployedge/microsoft-edge-policies#blockexternalextensions

* disabled
* unconfigured
* Not Configured
* Disabled
* enabled

also I'm looking for strings:
* ExtensionInstallForcelist
* ExtensionInstallSources
* ExtensionSettings

Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md]: closing this Outdated comment.  review latest wording/spelling/formatting of terms in Files Changed tab.

File: microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md (hunk 1 above)
Comment (cdwivedi1394) [reply microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md:155]: Mike, This looks good to me.

Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md:155]: i will resolve this thread after i delete the hidden comment that shows the orig draft

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**